src/pp_solvers/solver_mixin.py

```diff
@@ -236,9 +236,6 @@ def _initialize_linear_solver(self):
         if contact_transform is not None or thermal_transform is not None:
             solver_factory = LinearTransformedScheme(
                 nd=self.nd,
-                contact_group=contact_ind,
```

Comment:
these parameters were not used inside, and the linear transformation class should not care about those details